### PR TITLE
Add links to Discussions, improve contributor guidance and DX

### PR DESCRIPTION
**Pull request by OndraM**
Newly introduced GitHub [Discussions](https://github.com/php-webdriver/php-webdriver/discussions) look like a nice place to provide community support (questions, feedback, ideas etc.), so lets point users there, instead of current (unused) Facebook group and GitHub issues.

Contributor guidance in CONTRIBUTING.md was also extended.

And I've also rearranged composer scripts for better DX (developer experience).

### CODE REVIEW

**Comment by OndraM** [comment]
I also prepared new issue templates with link to Discussions. 

You can have a look on preview in my fork here: https://github.com/OndraM/php-webdriver/issues/new/choose

(I will send the PR with issue templates right after this PR is merged).

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -5,56 +5,85 @@ We love to have your help to make php-webdriver better!
...
 
-### Check coding style
+```sh
+export BROWSER_NAME="htmlunit" # see below for other browsers
+java -jar selenium-server-standalone-X.X.X.jar -log selenium.log &
+php -S localhost:8000 -t tests/functional/web/ &
+# Use following to run both unit and functional tests
+composer all
+# Or this to run only functional tests:
+./vendor/bin/phpunit --testsuite functional
```
Comment (williamdes) [inline .github/CONTRIBUTING.md:71]:
```suggestion
composer test -- --testsuite functional
```

**Review by williamdes — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (OndraM) [reply .github/CONTRIBUTING.md]: Nice, thanks!

File: tests/functional/WebDriverTestCase.php [hunk 2]
```diff
@@ -39,7 +39,9 @@ protected function setUp(): void
             if (getenv('BROWSER_NAME')) {
                 $browserName = getenv('BROWSER_NAME');
             } else {
-                $browserName = WebDriverBrowserType::HTMLUNIT;
+                $this->markTestSkipped(
```
Comment (williamdes) [inline tests/functional/WebDriverTestCase.php:42]: here you could invert the condition and call markTestSkipped because it throws and would stop execution

**Review by williamdes — APPROVED** [review]
Very good improvement of the composer commands !

File: tests/functional/WebDriverTestCase.php (hunk 2 above)
Comment (OndraM) [reply tests/functional/WebDriverTestCase.php:41]: True, thanks

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -5,56 +5,85 @@ We love to have your help to make php-webdriver better!
...
 
-Your code-style should comply with [PSR-2](http://www.php-fig.org/psr/psr-2/). To make sure your code matches this requirement run:
+If you want to run tests in different browser then "htmlunit" (Chrome or Firefox), you need to setup the browser driver (Chromedriver/Geckodriver), as it is [explained in wiki](https://github.com/php-webdriver/php-webdriver/wiki/Chrome)
+and then the `BROWSER_NAME` environment variable:
 
-    composer codestyle:check
+```sh
+...
+export BROWSER_NAME="chrome"
+composer all
```
Comment (williamdes) [inline .github/CONTRIBUTING.md:80]: We could suggest one line commands ?
```suggestion
BROWSER_NAME="chrome" composer all
```

File: .github/workflows/php.yaml [hunk 4]
```diff
@@ -20,20 +20,13 @@ jobs:
...
+        run: composer preinstall
 
-      - name: Check composer.json
-        run: |
-          composer validate --strict
-          composer normalize --dry-run
+      - name: Lint
+        run: composer lint
 
       - name: Run checks
```
Comment (williamdes) [inline .github/workflows/php.yaml:28]:
```suggestion
      - name: Run analysis
```

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (OndraM) [reply .github/CONTRIBUTING.md:80]: I prefer the export variant, it is more readable for me :shrug:

**Pull request merged**